experiments-rtl/src/app/__tests__/click.test.tsx

```diff
@@ -25,7 +25,36 @@ test("fireEvent.click()", () => {
     </button>
   );
 
-  const button = screen.getByText("Click");
+  const button = screen.getByText('Click');
   fireEvent.click(button);
   expect(handleClick).toHaveBeenCalledTimes(1);
 });
+
+it.only('is wrapped by act', async () => {
```

Comment:
pls remove `.only`
```suggestion
it('is wrapped by act', async () => {
```

Reply:
I deleted the test, it was just an experiment I was conducting and I forgot to remove it